config/locales/models.ja.yml

```
@@ -1,215 +1,10 @@
-      over_x_years:
-        one: 1年以上
-        other: "%{count}年以上"
-      x_seconds:
-        one: 1秒
-        other: "%{count}秒"
-      x_minutes:
-        one: 1分
-        other: "%{count}分"
-      x_days:
-        one: 1日
-        other: "%{count}日"
-      x_months:
-        one: 1ヶ月
-        other: "%{count}ヶ月"
-      x_years:
-        one: 1年
-        other: "%{count}年"
-    prompts:
-      second: 秒
-      minute: 分
-      hour: 時
-      day: 日
-      month: 月
-      year: 年
-  errors:
-    format: "%{attribute}%{message}"
-    messages:
-      accepted: を受諾してください
-      blank: を入力してください
-      confirmation: と%{attribute}の入力が一致しません
-      empty: を入力してください
-      equal_to: は%{count}にしてください
-      even: は偶数にしてください
-      exclusion: は予約されています
-      greater_than: は%{count}より大きい値にしてください
-      greater_than_or_equal_to: は%{count}以上の値にしてください
-      inclusion: は一覧にありません
-      invalid: は不正な値です
-      less_than: は%{count}より小さい値にしてください
-      less_than_or_equal_to: は%{count}以下の値にしてください
-      model_invalid: 'バリデーションに失敗しました: %{errors}'
-      not_a_number: は数値で入力してください
-      not_an_integer: は整数で入力してください
-      odd: は奇数にしてください
-      other_than: は%{count}以外の値にしてください
-      present: は入力しないでください
-      required: を入力してください
-      taken: はすでに存在します
-      too_long: は%{count}文字以内で入力してください
-      too_short: は%{count}文字以上で入力してください
-      wrong_length: は%{count}文字で入力してください
-    template:
-      body: 次の項目を確認してください
-      header:
-        one: "%{model}にエラーが発生しました"
-        other: "%{model}に%{count}個のエラーが発生しました"
-  helpers:
-    select:
-      prompt: 選択してください
-    submit:
-      create: 登録する
-      submit: 保存する
-      update: 更新する
-  number:
-    currency:
-      format:
-        delimiter: ","
-        format: "%n%u"
-        precision: 0
-        separator: "."
-        significant: false
-        strip_insignificant_zeros: false
-        unit: 円
-    format:
-      delimiter: ","
-      precision: 3
-      separator: "."
-      significant: false
-      strip_insignificant_zeros: false
-    human:
-      decimal_units:
-        format: "%n %u"
-        units:
-          billion: 十億
-          million: 百万
-          quadrillion: 千兆
-          thousand: 千
-          trillion: 兆
-          unit: ''
-      format:
-        delimiter: ''
-        precision: 3
-        significant: true
-        strip_insignificant_zeros: true
-      storage_units:
-        format: "%n%u"
-        units:
-          byte: バイト
-          eb: EB
-          gb: GB
-          kb: KB
-          mb: MB
-          pb: PB
-          tb: TB
-    percentage:
-      format:
-        delimiter: ''
-        format: "%n%"
-    precision:
-      format:
-        delimiter: ''
-  support:
-    array:
-      last_word_connector: "、"
-      two_words_connector: "、"
-      words_connector: "、"
-  time:
-    am: 午前
-    formats:
-      default: "%Y年%m月%d日(%a) %H時%M分%S秒 %z"
-      long: "%Y/%m/%d %H:%M"
-      short: "%m/%d %H:%M"
-    pm: 午後
+  books:
+    index:
+      title: "%{name}一覧"
```

`books.index.xxx`というキーだと本の画面以外では使われないので、

```suggestion
      title: "本一覧"
```

でもいいですね。

`"%{name}一覧"`を残すなら汎用性のあるキーを与えるのが良いと思います。